### PR TITLE
Lesson1

### PR DESCRIPTION
**Pull request by VeronikaVasilchikova**
**Date: 2020/02/27**

Here is my first home task in WA.
I added the following Webix components on the page: 
- Toolbar with Label and Button;
- List, Datatable and Form;
- Template with text and HTML-fragment.
I added additional styles to List, Button (that is situated below List) and Template.

I have a question: how to set height of medium row better ?
I would be greatful if you check my task and write your commets.

### CODE REVIEW

**Comment by ByTiger** [comment]
Using "autoConfig" and "columns" at the same time in "datatable" doesn't make sence.
It's better to use "const" instead of "let" whenever it possible.
Please, try to keep the same formatting in local places (e.g. autoConfig:true, scroll:"y").

File: index.html [hunk 1]
```diff
@@ -3,10 +3,136 @@
...
+                  {view:"button", value:"Add new", css:"webix_primary", width: 110},
+                  {view:"button", value:"Clear", width: 110},
+                ]
+              },
+            ]
+          };
+
+          webix.ui({
+            rows:[toolbar,
+              {height:500, cols:[menu, {view:"resizer"}, datatable, form]},
```
Comment (Zwillinge) [inline index.html]: the application is not full screen due to fixed height
think of adding a spacer to form elements

File: index.html [hunk 2]
```diff
@@ -3,10 +3,136 @@
...
+            css:"gray_menu",
+            rows:[
+              {
+                view:"list",
+                template:"#value#",
+                data:menu_data,
+                scroll:false,
+                css:"webix_layout_clean"
+              },
+              {
```
Comment (Zwillinge) [inline index.html]: this is a simple label reflecting state, so there's no need to use a button
using a template containing a span with an appropriate CSS class is better here

File: index.html (hunk 1 above)
Comment (Zwillinge) [reply index.html]: I see that recdent commit fixes it

**Pull request merged**